src/Javis.jl

```diff
@@ -424,6 +424,28 @@ for func in names(Luxor; imported = true)
     end
 end
 
+# shorthand declarations
+# Line
+include("shorthands/JLine.jl")
+
+# Circle
+include("shorthands/JCircle.jl")
+
+# Rect
+include("shorthands/JRect.jl")
+
+# Box
+include("shorthands/JBox.jl")
+
+# Ellipse
+include("shorthands/JEllipse.jl")
+
+# Star
+include("shorthands/JStar.jl")
+
+# Poly
+include("shorthands/JPoly.jl")
```

Comment:
I would remove the comments here besides the `#shorthand declarations` one as they don't provide any value.